src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
```

Comment:
Use semantic tags where possible. Here, you should use the `<div>` tag as a container for the logo link, as the `<header>` tag should be used for containing navigational links and other information typically found at the top of the page.